def.cf

```diff
@@ -150,6 +150,11 @@ bundle common def
         "$(sys.workdir)/httpd/logs/error_log", # Mission Portal
       };
 
+      "max_client_history_size" -> { "cf-hub", "CFEngine Enterprise" }
+        int => "50M",
```

Comment:
According to [the language concepts documentation on variables](https://docs.cfengine.com/latest/guide-language-concepts-variables.html)
suffixing an integer value with 'M' will multiply the value by 1024.

#+beign_quote
  Since computing systems such as storage and memory are based on
  binary values, CFEngine also provide the following uppercase
  suffixes to create integer values for common powers of 1024.

    'K' = value times 1024.
    'M' = value times 10242
    'G' = value times 10243
#+end_quote